CHANGELOG.rst

```diff
@@ -16,6 +16,12 @@ Current Developments
 
 **Security:** None
 
+v0.2.8
+====================
```

Comment:
Actually, you should leave this under the `Current Developments` since it isn't part of a release yet.  The version number is filled in automatically here by the `release.xsh` script.  Would you mind fixing this up, please?